test/e2e/performanceprofile/functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -168,6 +172,43 @@ var _ = Describe("[rfe_id:OCP-38968][ppc] Performance Profile Creator", func() {
 		})
 	})
 
+	It("[test_id:XXXX1] Verify PPC fails when offlined cpu count is more than available cpus", func() {
```

Comment:
There is no "test_id" for these tests.
They are really simple tests, but I have left this here just to raise the question: Should I ask for a test_id or just ignore the tag?

Reply:
let's start ignoring the tag, we can add it later

Reply:
done